x-pack/test/functional/apps/endpoint/endpoints_page.ts

```diff
@@ -1,45 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
- */
-
-import expect from '@kbn/expect';
-import { FtrProviderContext } from '../../ftr_provider_context';
-
-export default ({ getPageObjects, getService }: FtrProviderContext) => {
-  const pageObjects = getPageObjects(['common', 'endpoint']);
-  const browser = getService('browser');
-  const testSubjects = getService('testSubjects');
-  const esArchiver = getService('esArchiver');
-
-  describe('Endpoints List Page', function() {
-    this.tags(['skipCloud']);
-    before(async () => {
-      esArchiver.load('endpoint/endpoints');
```

Comment:
The data load is here, I don't see another one.  How is the test data being loaded?